nasa-gibs/worldview
# WV-3161 GHGC Layer

## Description

> Investigate how to integrate Greenhouse Gas Center data products into Worldview and implement one layer for initial MVP

## How To Test

1. `git clone WV-3161-GGC-layer`
2. `npm run build`
3. Open this [link](http://localhost:3000/?v=-185.12748508829483,-133.15312110045153,218.85413243710227,98.99603758167768&l=GHGC_Net_Primary_Produnction(opacity=0.5),Land_Water_Map&lg=true&t=2024-05-11-T11%3A58%3A15Z)
4. Ensure that the "Net Primary Production" layer functions properly

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
